File: Problems/1689_Partitioning_Into_Minimum_Number_Of_Deci-Binary_Numbers.cpp
#include<string>

using namespace std;
class Solution {
public:
    int minPartitions(string n) {
        int max_digit = 0;
        for(auto c: n){
            if(c-'0'>max_digit) max_digit = c-'0';
        }
        return max_digit;
    }
};

File: Problems/1689_Partitioning_Into_Minimum_Number_Of_Deci-Binary_Numbers.py
class Solution:
    def minPartitions(self, n: str) -> int:
        return max(int(c) for c in n)
# class Solution:
#     def minPartitions(self, n: str) -> int:
#         max_num = 0
#         for c in n:
#             max_num = max(max_num, int(c))
#         return max_num